File: desktop-app/src/main/java/entities/Discipline.java
package entities;

public class Discipline {
    private int id;
    private String name;
    private String description;
    private String weightClass;
    private int roundDuration;
    private int maxRounds;

    public Discipline() {
    }

    public Discipline(String name, String description, String weightClass,
            int roundDuration, int maxRounds) {
        this.name = name;
        this.description = description;
        this.weightClass = weightClass;
        this.roundDuration = roundDuration;
        this.maxRounds = maxRounds;
    }

    public Discipline(int id, String name, String description, String weightClass,
            int roundDuration, int maxRounds) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.weightClass = weightClass;
        this.roundDuration = roundDuration;
        this.maxRounds = maxRounds;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getWeightClass() {
        return weightClass;
    }

    public void setWeightClass(String weightClass) {
        this.weightClass = weightClass;
    }

    public int getRoundDuration() {
        return roundDuration;
    }

    public void setRoundDuration(int roundDuration) {
        this.roundDuration = roundDuration;
    }

    public int getMaxRounds() {
        return maxRounds;
    }

    public void setMaxRounds(int maxRounds) {
        this.maxRounds = maxRounds;
    }

    @Override
    public String toString() {
        return "Discipline{id=" + id + ", name='" + name + "', weightClass='" + weightClass + "', roundDuration="
                + roundDuration + ", maxRounds=" + maxRounds + "}";
    }
}

File: smartfight/src/Entity/Discipline.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity]
#[ORM\Table(name: 'discipline')]
class Discipline
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column(type: 'integer')]
    private ?int $id = null;

    #[ORM\Column(type: 'string', length: 100, unique: true)]
    private string $name;

    #[ORM\Column(type: 'text', nullable: true)]
    private ?string $description = null;

    #[ORM\Column(name: 'weight_class', type: 'string', length: 60, nullable: true)]
    private ?string $weightClass = null;

    #[ORM\Column(name: 'round_duration', type: 'integer', options: ['default' => 3])]
    private int $roundDuration = 3;

    #[ORM\Column(name: 'max_rounds', type: 'integer', options: ['default' => 3])]
    private int $maxRounds = 3;

    #[ORM\Column(name: 'created_at', type: 'datetime')]
    private \DateTimeInterface $createdAt;

    #[ORM\Column(name: 'updated_at', type: 'datetime')]
    private \DateTimeInterface $updatedAt;

    public function getId(): ?int { return $this->id; }
    public function getName(): string { return $this->name; }
    public function __toString(): string { return $this->name; }
}
